gradle/scripts/dependency.gradle

```diff
@@ -30,10 +30,11 @@ ext.externalDependency = [
     "festAssert"          : "org.easytesting:fest-assert-core:2.0M10",
     "systemRules"         : "com.github.stefanbirkner:system-rules:1.16.0",
 
-    "play_java_jdbc"      : "com.typesafe.play:play-java-jdbc_2.11:2.4.11",
```

Comment:
Why do we want to downgrade to 2.4.6, from 2.4.11?

Reply:
Artifactory only has 2.4.6